consent-ws/src/main/java/org/broadinstitute/consent/http/resources/DataRequestReportsResource.java

```diff
@@ -79,4 +89,34 @@ public Response downloadReviewedDARs() {
             return createExceptionResponse(e);
         }
     }
+
+    private Boolean requiresManualReview(Document dar) throws IOException {
+        Map<String, Object> form = parseAsMap(dar.toJson());
```

Comment:
Not for this PR, but we should keep this in mind for future work. We parse this as a map in lots of places, looking for particular values. I think this kind of code should be in a utility class specifically designed around parsing out `dar` documents.